lte/cloud/go/services/lte/servicers/builder_servicer.go

```diff
@@ -602,3 +606,16 @@ func getNetworkProbeConfig(networkID string) ([]*lte_mconfig.NProbeTask, *lte_mc
 	}
 	return npTasks, liUes
 }
+
+func getNetworkSentryConfig(network *configurator.Network) *lte_mconfig.SentryConfig {
+	sentryConfig, found := network.Configs[orc8r.NetworkSentryConfig].(*models.NetworkSentryConfig)
```

Comment:
super minor nit, but I think we use `found` for map look up and `ok` for interface conversion/casting? 

So I think in other parts of the code it looks like
```
iSentryConfig, found := network.Configs[orc8r.NetworkSentryConfig]
// check found error
sentryConfig, ok := iSentryConfig.(*models.NetworkSentryConfig)
// check ok error
```

My go is a bit rusty, does this line work even if `network.Configs[orc8r.NetworkSentryConfig]` is nil?

Reply:
We can also add a unit test for a network that doesn't have this sentry config to test that theory out I suppose :)

Reply:
`sentryConfig, found := network.Configs[orc8r.NetworkSentryConfig].(*models.NetworkSentryConfig)` does work and `found` is `false` when the key isn't found. But I have changed it to handle the key lookup and conversion errors separately because that's how they're handled in other places.

There are existing tests, which I did not modify, in `builder_servicer_test.go` that have the SentryConfig set to nil. So no need for a new unit test.